File: C++/EX_condicional/ex1_menorde3/menorde3.cpp
/*Exemplo 1: 
Primeiro valor: 7
Segundo valor: 3
Terceiro valor: 8
MENOR = 3 */

#include<bits/stdc++.h>

using namespace std;

int main(){
    
    int num1, num2, num3, menor;

    cout << "Primeiro valor: ";
    cin >> num1;

    cout << "Segundo valor: ";
    cin >> num2;

    cout << "Terceiro valor: ";
    cin >> num3;

    cout << endl;
    if (num1 < num2 && num1 < num3){
        menor = num1;

    } else if (num2 < num3) {
        menor = num2;

    } else {
        menor = num3;
    }

    cout << "MENOR = " << menor << endl;
    


    return 0;
}

File: Python/Ex estrutura condicional/ex1 menorde3/menorde3.py
"""
Exemplo 1: 
Primeiro valor: 7
Segundo valor: 3
Terceiro valor: 8
MENOR = 3
"""

valor1 = int(input("Primeiro valor: "))
valor2 = int(input("Segundo valor: "))
valor3 = int(input("Terceiro valor: "))

if (valor1 < valor2 and valor1 < valor3):
    menor = valor1
elif (valor2 < valor3):
    menor = valor2
else:
    menor = valor3

print()
print(f"MENOR = {menor}")
